### ISSUE TITLE
Annotations do not show

### ISSUE THREAD

**Issue by hbs** [cityzendata/warp10-quantum#48]
The following WarpScript should display annotations in Quantum but doesn't in 3.0.0-rc1

NEWGTS
1 20
<%
3 m * NOW SWAP - NaN NaN NaN 'Annotation' ADDVALUE
%> FOR

**Comment by Giwi** [comment]
Ok I've found the problem : 

```
Uncaught ReferenceError: d3 is not defined
    at HTMLElement.contentChanged (quantum-plot.html:50)
```

**Comment by Giwi** [comment]
shipped in 3.0.0-rc5, deployed in preview
